auraed/tests/cells_list.rs

```diff
@@ -0,0 +1,199 @@
+use client::cells::cell_service::CellServiceClient;
```

Comment:
nitpick: the service is called `cell_service`, so maybe we should call the test `cell_list`?